go.work.sum

```diff
@@ -284,54 +297,42 @@ gopkg.in/square/go-jose.v2 v2.5.1 h1:7odma5RETjNHWJnR32wx8t+Io4djHE1PqxCFx3iiZ2w
 honnef.co/go/tools v0.2.2 h1:MNh1AVMyVX23VUHE2O27jm6lNj3vjO5DexS4A1xvnzk=
 k8s.io/api v0.24.2 h1:g518dPU/L7VRLxWfcadQn2OnsiGWVOadTLpdnqgY2OI=
 k8s.io/api v0.24.2/go.mod h1:AHqbSkTm6YrQ0ObxjO3Pmp/ubFF/KuM7jU+3khoBsOg=
-k8s.io/api v0.25.0-rc.0 h1:JzQmjOfEAkOyCzDyFTayWB3nCQeS4TcCldrl2Q+417g=
-k8s.io/api v0.25.0-rc.0/go.mod h1:H8qkBzdcUEFJEhgA14ip3PpzUFN0XXPgnlDnu/KaDf0=
 k8s.io/apimachinery v0.24.2 h1:5QlH9SL2C8KMcrNJPor+LbXVTaZRReml7svPEh4OKDM=
 k8s.io/apimachinery v0.24.2/go.mod h1:82Bi4sCzVBdpYjyI4jY6aHX+YCUchUIrZrXKedjd2UM=
-k8s.io/apimachinery v0.25.0-rc.0 h1:k3n1k01fFHrTsRHM8xt+z9yOpak7KmO/pbmD0/tAkzs=
-k8s.io/apimachinery v0.25.0-rc.0/go.mod h1:qMx9eAk0sZQGsXGu86fab8tZdffHbwUfsvzqKn4mfB0=
 k8s.io/apiserver v0.24.2 h1:orxipm5elPJSkkFNlwH9ClqaKEDJJA3yR2cAAlCnyj4=
 k8s.io/apiserver v0.24.2/go.mod h1:pSuKzr3zV+L+MWqsEo0kHHYwCo77AT5qXbFXP2jbvFI=
-k8s.io/apiserver v0.25.0-rc.0 h1:PCLYM37Cq9SEbb6jFhA1E7NpfW6GddG2AzgECMU5jlw=
```

Comment:
should not be comitted? idk... it feels weird it now removes 0.25... this file is weird :-D

Reply:
Yes I wasn't sure. It keeps changing on my local and I'm unsure if I should commit it. I just assumed that Go knows what it is doing...

Reply:
yeah, same change for me, so let's commit it ;)

Reply:
ok, pushed one more time, bringing it back.